Shopify/shopify_app
# Add new declarative webhook generators

### What this PR does
This PR adds a new generator to add the neccessary controller, job and routes for apps using webhook subscriptions defined in the `shopify.app.toml` file.

### Reviewer's guide to testing
* Use the generator to add the new files to a development app.
* Confirm that you are able to successfully receive and process webhook events.

This PR depends on this [PR](https://github.com/Shopify/shopify-app-template-ruby/pull/138).

### Things to focus on

### Checklist

Before submitting the PR, please consider if any of the following are needed:
